user-registration-app/package.json

```diff
@@ -12,60 +12,56 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^7.2.6",
-    "@angular/common": "^7.2.6",
-    "@angular/compiler": "^7.2.6",
-    "@angular/core": "^7.2.6",
-    "@angular/forms": "^7.2.6",
-    "@angular/http": "^7.2.6",
-    "@angular/fire": "5.3.0",
-    "@angular/platform-browser": "7.2.6",
-    "@angular/platform-browser-dynamic": "^7.2.6",
-    "@angular/router": "^7.2.6",
-    "@ngx-progressbar/core": "5.0.1",
-    "@ngx-progressbar/http": "5.0.1",
-    "@ngx-progressbar/router": "5.0.1",
-    "address-validator": "0.2.2",
-    "ajv": "6.5.2",
+    "@angular/animations": "^11.0.2",
+    "@angular/common": "^11.0.2",
+    "@angular/compiler": "^11.0.2",
+    "@angular/core": "^11.0.2",
+    "@angular/fire": "^6.1.1",
+    "@angular/forms": "^11.0.2",
+    "@angular/http": "^7.2.16",
+    "@angular/platform-browser": "^11.0.2",
+    "@angular/platform-browser-dynamic": "^11.0.2",
+    "@angular/router": "^11.0.2",
+    "ajv": "^6.12.6",
     "angular2-materialize": "15.1.10",
-    "core-js": "^2.4.1",
-    "firebase": "7.24.0",
+    "core-js": "^3.7.0",
+    "firebase": "^8.1.1",
     "hammerjs": "2.0.8",
-    "jquery": "2.2.4",
-    "libphonenumber-js": "1.0.24",
-    "lodash": "4.17.10",
+    "jquery": "^3.5.1",
+    "libphonenumber-js": "^1.9.4",
+    "lodash": "4.17.20",
     "materialize-css": "^0.100.2",
     "ng-animate": "0.3.4",
-    "ngx-alerts": "3.0.0",
-    "ngx-pagination": "3.2.0",
+    "ngx-pagination": "^5.0.0",
+    "ngx-progressbar": "^6.0.3",
+    "ngx-toastr": "^13.1.0",
```

Comment:
The current alerts package isn't maintained anymore so I replaced it with one that is